<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>734</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddBankButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Bank</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetBankButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Banks List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddUserButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>570</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetUserButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>570</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Users List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenBankAccountButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>570</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Bank Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputLine">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OutputWindow">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
